std/digest/murmurhash.d

```diff
@@ -0,0 +1,789 @@
+/**
+Computes MurmurHash hashes of arbitrary data. MurmurHash is a non-cryptographic
+hash function suitable for general hash-based lookup.
+
+This module conforms to the APIs defined in $(D std.digest.digest).
+
+This module publicly imports $(D std.digest.digest) and can be used as a stand-alone module.
+
+Note: The current implementation is optimized for little endian architectures.
+It will exhibit different results on big endian architectures and a slightly less uniform distribution.
+
+License: $(WEB www.boost.org/LICENSE_1_0.txt, Boost License 1.0).
+Authors: Guillaume Chatelet
+References: $(LINK2 https://code.google.com/p/smhasher/wiki/MurmurHash3, Reference implementation)
+$(BR) $(LINK2 https://en.wikipedia.org/wiki/MurmurHash, Wikipedia on MurmurHash)
+*/
+module std.digest.murmurhash;
+
+public import std.digest.digest;
+
+@safe:
+
+///
+unittest
```

Comment:
DDox recognizes these two examples, but DDoc doesn't seem to. Maybe they need to be moved up?